lib/home/view_model/home_page_view_model.dart

```diff
@@ -100,8 +101,8 @@ class HomePageViewModel extends BaseViewModel {
 
   /// A stream which contains all previously finished sessions
   void _subscribeToPastSessionsStream() {
-    StreamSubscription subscription = _pastSessionsRepository.pastSessionsStream
-        .map((event) => event.length)
+    StreamSubscription subscription = _meditationRepository.meditationStream
```

Comment:
So we want to completely replace the middleware repo for getting the past sessions?
I thought we agreed on only using the local one as a fallback if we don't receive data with the request. 

So instead we might want to add the local repo to the past sessions repo and update the stream with the entries in shared preferences in the catch block of the network request. 

Also for now we could just store locally in the train method. This will supposedly be refactored in the near future though to use a POST method on a new endpoint